### DISCUSSION TITLE
React app vercel Failed to load resource: net::ERR_TIMED_OUT

### DISCUSSION THREAD

**Discussion by iadh-mindbridgexr** [vercel/community#5862] [Help]
### Summary

my deployment is not working

I have a react app deployed on vercel showing Failed to load resource: net::ERR_TIMED_OUT

I couldn't find this error anywhere.

The app gets successfully built and deployed

The app is showing in the preview

When accessing the domain link it is showing the above error.

https://mindbridgexr-dashboard.vercel.app/

I'm not using any special functions 

Please help

### Steps to Reproduce

1. access the provided URL
2. open developer tools
3. go to console

**Comment by ozanArslan2424** [comment]
Were you able to fix this? I'm having the same issue.

**Reply by SchemeSonic** [reply]
vercel.app, 27/03/2024 tarihli ve 490.05.01.2024.-207622 sayılı Bilgi Teknolojileri ve İletişim Kurumu kararıyla erişime engellenmiştir. http://internet.btk.gov.tr/sitesorgu/http://internet.btk.gov.tr/sitesorgu/

**Reply by SchemeSonic** [reply]
you can use vpn or assign custom domain to your vercel app

**Reply by ozanArslan2424** [reply]
bugün baktığımda böyle bir karar yok, daha sonra da böyle bir erişim engeli gelir mi? Hobi projesi için domain parası harcamak istemiyorum da...

**Reply by SchemeSonic** [reply]
Surekli bir kapanma olacagini dusunmuyorum, zaman zaman kisa sureli kapanmalar olabiliyor, aynisi netlify icin de gecerli. Hobi projeleri icin bir problem teskil etmez bu durum.
